apiserver/restrict_caasmodel.go

```diff
@@ -57,6 +57,16 @@ var commonModelFacadeNames = set.NewStrings(
 	"Undertaker",
 	"Uniter",
 	"VolumeAttachmentsWatcher",
+
+	// for caas controller.
+	"Provisioner",
+	"Resumer",
+	"MachineActions",
+	"ProxyUpdater",
+	"Machiner",
+	"ExternalControllerUpdater",
+	"Upgrader",
```

Comment:
Upgrader and FanConfigurer and MachineActions etc not applicable
Can we add a TODO to come back and fix. Also add a trello card
We'll also need a trello card to implement upgrades

Reply:
`FanConfigurer` seems required,  coz peergrouper etc needs it to make ip discoverable.
Sure, we can add a `TODO` to revisit here later.

Reply:
ok, a TODO is fine for now